newrelic/exporter_test.go

```diff
@@ -224,10 +224,10 @@ func TestEndToEndMeter(t *testing.T) {
 	instruments := map[string]data{
 		"test-int64-counter":    {metric.CounterKind, metric.Int64NumberKind, 1},
 		"test-float64-counter":  {metric.CounterKind, metric.Float64NumberKind, 1},
```

Comment:
Should add test cases for `UpDownCounter`, `SumObserver`, and `UpDownSumObserver`.

Reply:
Added test cases!